mustang_core/lib/src/store/mustang_store.dart

```diff
@@ -205,6 +214,31 @@ class MustangStore {
     }
   }
 
-  /// Returns persistence status of the store
-  static bool get persistent => _persistent;
+  /// Deletes all objects except the ones passed in to the the
+  /// preserveModels argument
+  static Future<void> deleteObjects({
+    required List<String> preserveModels,
+  }) async {
+    List<String> modelsToDelete = [];
+    _hashStore.removeWhere((type, _) {
+      bool preserve = preserveModels.contains(type);
+      if (!preserve) {
+        modelsToDelete.add(type);
+      }
+      return !preserve;
+    });
+    await deletePersistedState(modelsToDelete);
+  }
+
+  /// Returns the stringified model from the persistence store.
+  /// Returns null if the model does not exist.
+  static Future<String?> getPersistedObject(String modelName) async {
```

Comment:
This looks like a new method...do we need to read the stringified model values in the app?